support-frontend/assets/components/thankYou/thankYouModuleData.tsx

```diff
@@ -55,9 +66,15 @@ export const getThankYouModuleData = (
 	isOneOff: boolean,
 	amountIsAboveThreshold: boolean,
 	campaignCode?: string,
+	supportReminder?: ThankYouSupportReminderState,
+	feedbackSurveyHasBeenCompleted?: boolean,
 ): Record<ThankYouModuleType, ThankYouModuleData> => {
-	const { feedbackSurveyHasBeenCompleted, supportReminder } =
-		useContributionsSelector((state) => state.page.checkoutForm.thankYou);
+	const [feedbackSurveyCompleted, SetFeedbackSurveyCompleted] =
```

Comment:
The naming here is a little unclear.

It feels like we have three states:
* Initial state
* Default state
* Current state

Would renaming the values to e.g.
* `initialFeedbackSurveyHasBeenCompleted`
* `defaultFeedbackSurveyHasBeenCompleted`
* `feedbackSurveyHasBeenCompleted`

read a little easier?

Reply:
Not sure about `initialFeedbackSurveyHasBeenCompleted` as its either `defaultFeedbackSurveyHasBeenCompleted ` or `feedbackSurveyHasBeenCompleted` only. Seems quite a lot of code to replace a default to false. However, have made a change think lines up with my understanding of above - could you check have this right? Thanks.